static/js/admin.js

```diff
@@ -0,0 +1,16 @@
+var table = new Tabulator("#users-list", {
+    height: "70%",
+    data: adminData.users,
```

Comment:
I think it would be better for this line to something like:
`data: (adminData && adminData.users) || {}`
in case adminData is undefined